build.gradle

```diff
@@ -185,7 +195,7 @@ task prepareGameEngineProperties() {
     }
 }
 
-task allPlatform(type: Zip, group: 'release', dependsOn: [renameShadowJar, updateAssets, prepareGameEngineProperties]) {
```

Comment:
Are you sure the icon files are not needed by TripleA at runtime?

Reply:
#2082 should answer both of your questions from this review. :smile:

Reply:
Indeed. 
Would you mind submitting a PR that removes the obsolete icns file?

Reply:
triplea-game/assets#19